site/content/how-to/data-plane-configuration.md

```diff
@@ -112,3 +112,51 @@ Status:
 ```
 
 If everything is valid, the `ResolvedRefs` condition should be `True`. Otherwise, you will see an `InvalidParameters` condition in the status.
+
```

Comment:
I would update the overview at the beginning where it mentions the types of things you can set to include log level.